accepted/2022/dotnet-release-labels.md

```diff
@@ -153,16 +156,16 @@ A given presentation (like Visual Studio) can print long-form or alternate versi
 
 ## Affect on `releases.json`
```

Comment:
Should this be Effect instead of Affect?

Reply:
Geez. That English degree was a waste!

```suggestion
## Effect on `releases.json`
```